src/Domain/DonationRepository.php

```diff
@@ -151,6 +151,7 @@ public function buildFromApiRequest(DonationCreate $donationData): Donation
             $this->fundRepository->pullForCampaign($campaign);
 
             $this->getEntityManager()->flush();
+            $this->getEntityManager()->persist($campaign);
```

Comment:
@bdsl I think this is
1. not related to the stated issue?
2. probably redundant because `SalesforceReadProxyRepository::pull()` has:
`$this->getEntityManager()->persist($proxy);`

Reply:
Thanks, I must have left this change in by mistake when working on the other issue. Feel free to delete this line or I will when I get to my desk.